dbt/include/spark/macros/materializations/incremental/strategies.sql

```diff
@@ -67,7 +70,7 @@
     {#-- insert statements don't like CTEs, so support them via a temp view #}
     {{ get_insert_overwrite_sql(source, target) }}
   {%- elif strategy == 'merge' -%}
-  {#-- merge all columns with databricks delta - schema changes are handled for us #}
+  {#-- merge all columns with databricks delta or iceberg - schema changes are handled for us #}
```

Comment:
For Iceberg, schema changes are only automatic when the table (or write) has the  merge schema property set.

As this is an abstract API in Spark, it might be easier to just say “for data sources which implement merge into” or similar? WDYT?

Reply:
Or I suppose the table “format” might be consistent with how the code is written in some places.

Reply:
I am very much ok with updating this comment, the or iceberg was added only because it came along with my change.  I like the more generic explanation you offer perhaps maybe add databricks and iceberg as a "for example"?  Something along the lines of

{#-- merge all columns for datasources which implement  merge into (e.g. databricks, iceberg) - schema changes are handled for us #}